### DISCUSSION TITLE
Fine-tuning AraBERT for multi-label text classification

### DISCUSSION THREAD

**Discussion by NoraAlt** [aub-mind/arabert#67] [Q&A] [answered]
Hello,

I am following the example [here](https://colab.research.google.com/drive/1P9iQHtUH5KUbTVtp8B4-AopZzEEPE0lw?usp=sharing), to fine-tune AraBERT for **Multi-label Classification**.

The dataset looks like:
<img width="928" alt="shot" src="https://user-images.githubusercontent.com/31368075/107872620-7887ba00-6ebc-11eb-818c-1b030a5af05c.png">

`Label_list=['Optimistic','Thankful','Empathetic','Pessimistic','Anxious','Sad','Annoyed','Denial','Official','Surprise','Joking']`

I could not figure out how to pass the labels.. Do I need to do one hot encoding of the labels?

Any ideas?
Would greatly appreciate your help!

**Comment by WissamAntoun** [comment, marked as answer]
I don't think this notebook works for your case. But here is an old notebook that implements multi-label classification with BERT. https://github.com/abhimishra91/transformers-tutorials/blob/master/transformers_multi_label_classification.ipynb

you need to modify it a bit to add the AraBERT preprocessing.
